docs/dapp/sapphire/browser.md

```diff
@@ -0,0 +1,188 @@
+---
+description: Writing Sapphire dApp for browser and Metamask
+---
+
+# Browser Support
+
+Confidential Sapphire dApps work in web browsers by wrapping the
+Metamask extension for signing and encrypting transactions.
```

Comment:
```suggestion
Ethereum provider to enable signing and encryption calls and transactions.
```
calls may be signed
txs are already signed
both may be encrypted

Reply:
> both may be encrypted

Can a user or dapp control whether the tx will be encrypted or not?

Reply:
Looking at https://github.com/oasisprotocol/sapphire-paratime/blob/main/clients/js/src/compat.ts the envelope is always encrypted, if the cipher object is not overriden right? Perhaps we should add this information (encrypted yes/no) as EIP712 attribute?